test/ITF1788_tests/libieeep1788_tests_elem.jl

```diff
@@ -5018,4 +5018,20 @@ end
     @test max(DecoratedInterval(Interval(3.0, 3.5), com), DecoratedInterval(Interval(2.0, 4.0), com)) == DecoratedInterval(Interval(3.0, 4.0), com)
     @test decoration(max(DecoratedInterval(Interval(3.0, 3.5), com), DecoratedInterval(Interval(2.0, 4.0), com))) == decoration(DecoratedInterval(Interval(3.0, 4.0), com))
 end
+
+@testset "minimal_sinpi_test" begin
```

Comment:
Did you add these by hand or do they come from the ITF1788 test suite?

If you added them by hand, they should *not* go in this file, since it will be regenerated automatically later.

Reply:
Ok, let me change the location of tests.